src/crypto/sign/ed25519.rs

```diff
@@ -477,7 +477,7 @@ mod test {
         let mut creation_state = State::init();
         creation_state.update(&m);
 
-        let creation_state_cloned = creation_state.clone();
+        let creation_state_cloned = creation_state;
```

Comment:
This should have a different name

Reply:
Wow. The name of the test is `test_streaming_clone`. So it tests whether we can clone it. I will rename it to `test_streaming_copy` and `creation_state_cloned` -> `creation_state_copy`. How does it sound?

Reply:
Ah. There is already such a test `test_streaming_copy`.

Reply:
I removed the test for clone()